### ISSUE TITLE
Unable to render anything

### ISSUE THREAD

**Issue by ancashoria** [researchgate/react-intersection-list#126]
The developer experience using this component is terrible. I was unable to get it to work because of the following error.

Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of 'Sentinel'.

Basically it breaks when it renders the sentinel item. 

The props I have:

```javascript
<List
        items={data}
        itemsRenderer={(items, ref) => {
          return <div className="list" ref={ref}>{items}</div>
        }}
        renderItem={(index, key) => {
          const itemData = data[index]
          return <div key={itemData.id}>{itemData.name}</div>
        }} />
```

**Comment by danez** [comment]
This was fixed by #124 and is in version 3.0.10.

**Comment by ancashoria** [comment]
Thanks for the heads up.
